src/main/java/aquality/selenium/core/utilities/IUtilitiesModule.java

```diff
@@ -0,0 +1,27 @@
+package aquality.selenium.core.utilities;
+
+/**
+ * Provides implementations for utilities module.
+ */
+public interface IUtilitiesModule {
+
+    /**
+     * @return implementation of {@link IElementActionRetrier}.
+     */
+    default Class<? extends IElementActionRetrier> getElementActionRetrierImplementation() {
+        return ElementActionRetrier.class;
+    }
+
+    /**
+     * Provides default {@link ISettingsFile} with settings.
+     * Default value is settings.json.
+     * You are able to override this path, by setting environment variable 'profile'.
+     * In this case, settings file will be settings.{profile}.json.
+     *
+     * @return An instance of settings.
+     */
+    default ISettingsFile getSettingsFileImplementation() {
```

Comment:
that's not an implementation but instance. please rename it according to return value